### DISCUSSION TITLE
My account got suspended

### DISCUSSION THREAD

**Discussion by fzn0x** [forem/forem#17815] [Q&A] [answered]
This is really strange, I was writing a post about work culture https://fzn0x.bearblog.dev/id-sisi-gelap-scbd-dan-pengalaman-saya-bekerja-di-scbd/ and after that get 
![Capture](https://user-images.githubusercontent.com/50759463/171705076-919d03f8-d616-4b81-968f-41fbb40623e3.PNG)
this kind of suspend on my account

**Comment by fzn0x** [comment, marked as answer]
Resolved by internal discussion with dev.to developers

**Comment by nmendybayev** [comment]
@fzn0x How did you resolve the problem? I have the same one.

**Reply by fzn0x** [reply]
Send them email, ask for unban

**Reply by nmendybayev** [reply]
I did it like 3 times. 3 weeks went still no answer).

**Reply by fzn0x** [reply]
try to contact this email `yo@dev.to`  @nmendybayev 

![Screenshot 2024-06-29 150441](https://github.com/forem/forem/assets/50759463/1cfaebe6-d655-4671-b2e4-15ff6f8f13d9)

and you will got response:

![Screenshot 2024-06-29 150531](https://github.com/forem/forem/assets/50759463/a54ed493-3648-4b6a-9dc5-7be6b4a37199)
